proto/asn1-pdu-proto/README.md

```diff
@@ -0,0 +1,5 @@
+# ASN.1 PDU Protobuf
+This protobuf represents an ASN.1 PDU defined in spec X.690 (2015). Along with the converter and LPM, the protobuf can be used to generate intersting inputs for a fuzzer.

```

Comment:
```suggestion
This [protobuf](https://developers.google.com/protocol-buffers) represents a
DER-encoded ASN.1 PDU, defined in X.690 (2015). When used with
[libprotobuf-mutator](https://github.com/google/libprotobuf-mutator), the protobuf can be used
to produce valid DER sequences, as well as syntactically invalid (e.g. BER) and structurally
invalid (e.g. random data) inputs for a fuzzer.
```

The motivation behind these changes are:
* The Protobuf largely reflects the DER-encoding. There are other encodings (e.g. XER, CER, PER)
   which this doesn't fully touch on, and this isn't a full X.680 AST.
* This is Markdown, so we can linkify things like LPM :) 
* Calling out some of the capabilities here - it's DER, sometimes BER (e.g. indefinite encodings),
  and sometimes just invalid (e.g. the random data)
* Wrap to 78/80